src/_node-compat/headers.ts

```diff
@@ -275,3 +293,14 @@ function _normalizeValue(
   }
   return typeof value === "string" ? value : String(value ?? "");
 }
+
+function isBannedHeader(name: string): boolean {
```

Comment:
This logic is specific to http2 support only. We should seperate by exntending base classes or make new ones for http2.

Reply:
(seems we still need to move this?) PS this check could be simplified to `name[0] === ":"` inlined in places needed it for all psudo headers

Reply:
Yes, work is in progress :) Just pushed other changes